tensorbay/cli/config.py

```diff
@@ -11,6 +11,8 @@
 
 from .utility import error, is_accesskey, read_config, update_config, write_config
 
+LEGAL_KEYS = {"editor", "timeout", "is_internal", "max_retries"}
```

Comment:
Is it needed elsewhere? If not, make it private.

Reply:
ok~